src/calculator.js

```diff
@@ -1,48 +1,23 @@
+module.exports = exports;
 exports._check = () => {
-  // DRY up the codebase with this function
-  // First, move the duplicate error checking code here
-  // Then, invoke this function inside each of the others
-  // HINT: you can invoke this function with exports._check()
-};
-
-exports.add = (x, y) => {
+  let x;
+  let y;
   if (typeof x !== 'number') {
     throw new TypeError(`${x} is not a number`);
   }
   if (typeof y !== 'number') {
     throw new TypeError(`${y} is not a number`);
   }
+}
+exports.add = (x, y) => {
```

Comment:
Unexpected block statement surrounding arrow body.

###### See http://eslint.org/docs/rules/arrow-body-style for details about this rule